Makefile

```diff
@@ -288,23 +296,27 @@ dist_manifest_in_files := \
     README.md \
     README_PYPI.md \
     requirements.txt \
-    *.py \
+    pyproject.toml \
     $(package_name)/*.py \
     $(package_name)/*/*.py \
     $(package_name)/*/*/*.py \
 
 # Files that are dependents of the distribution archive.
 # Keep in sync with dist_manifest_in_files.
-dist_dependent_files := \
+dist_dependent_files_all := \
     LICENSE.txt \
     README.md \
     README_PYPI.md \
     requirements.txt \
-    $(wildcard *.py) \
+		pyproject.toml \
     $(wildcard $( package_name)/*.py) \
     $(wildcard $(package_name)/*/*.py) \
     $(wildcard $(package_name)/*/*/*.py) \
 
+# The actually used dependency list, which removes the version file. Reason is that the
```

Comment:
slight english grammar thing. Change to "The dependency list actually used."

Reply:
Changed as proposed.